notebooks/calculate_sensitivity_from_Crab.ipynb

```diff
@@ -0,0 +1,2032 @@
+{
```

Comment:
<p>you can merge this cell with the one above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>